Gallery/Features/index.scss

```diff
@@ -1,6 +1,22 @@
-@import './styles/Settings/all.scss';
-@import './styles/Tools/all.scss';
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+@import '~DolittleStyles/Settings/all.scss';
+@import '~DolittleStyles/Tools/all.scss';
 
+main {
+    &.navigation-left {
```

Comment:
Health warning; not being in the context I don't see the entire picture. But, is class something that is required from consumers of the navigation componet?

Reply:
Yes, this is something we haven't solved yet. Back to what we have talked about regarding global settings in the final output. The composition in the consumption app, needs styling to avoid the components from crashing and overlapping. I hope these kind of issues will be solved in the layout components.

Reply:
You don't HAVE to decorate with a class. This gallery app needs that to be able to show different variations of the navigationview.